nats/src/natsprov.rs

```diff
@@ -102,8 +102,14 @@ fn create_subscription(
                     let buf = serialize(&dm).unwrap();
 
                     let d = dispatcher.read().unwrap();
-                    if let Err(e) = d.dispatch(&actor, OP_DELIVER_MESSAGE, &buf) {
+                    match d.dispatch(&actor,OP_HANDLE_MESSAGE, &buf){
+                      Ok(buf) =>{
+                        let broker_msg:BrokerMessage = deserialize(&buf).unwrap();
+                        msg.respond(broker_msg.body)?;
```

Comment:
Per other comment, this contract _does not mandate_ a response from the actor. It needs to remain a void operation.